tests/testthat/reporters/debug.txt

```diff
@@ -50,10 +50,13 @@
 1: tests.R#72: test_that("Skip:3", {
 })
 
-1: tests.R#78: warning("abc")
+1: tests.R#77: skip_if(condition)
+2: skip.R#86: skip(message)
```

Comment:
This is the line that produced different output in `R CMD CHECK`. That showed only

`2: skip(message)`
rather than
`2: skip.R#86: skip(message)`

I'm still not sure what the bug is, but I think it may be an issue in `recover2`, which seems out of scope for this PR.

Reply:
I don't see this failure, either locally or on travis?

Reply:
Oh hmmm, seems like there's some problem with the reporter tests. I'll look into it

Reply:
Maybe this is #851?

Reply:
Ok, now I see the problem

Reply:
For now the easiest thing is just to remove that new skip. There are easier ways to test the output.